Lib/typing.py

```diff
@@ -1659,16 +1659,18 @@ def close(self): ...
     cls._is_runtime_protocol = True
     return cls
 
+try:
+    from _typing import cast
+except ImportError:
+    def cast(typ, val):
+        """Cast a value to a type.
 
-def cast(typ, val):
-    """Cast a value to a type.
-
-    This returns the value unchanged.  To the type checker this
```

Comment:
With this PR the docstrings is lost so `help(cast)` will not show the same thing. Please ensure that the clinic code and the fallback function have the same docstring as before

Reply:
Fixed, thanks for pointing on it.